make/modules/java.base/Lib.gmk

```diff
@@ -80,6 +80,22 @@ $(eval $(call SetupJdkLibrary, BUILD_LIBNIO, \
     NAME := nio, \
     OPTIMIZATION := HIGH, \
     WARNINGS_AS_ERRORS_xlc := false, \
+    Net.c_CFLAGS := -I$(OPENJ9OMR_TOPDIR)/include_core \
+        -I$(OUTPUTDIR)/vm/runtime/jcl, \
```

Comment:
Please put the first option on the next line (like is done for `nio_util.c_CFLAGS`).